packages/schematics/angular/application/files/browserslist.template

```diff
@@ -1,11 +1,15 @@
-# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
 # For additional information regarding the format and rule options, please see:
 # https://github.com/browserslist/browserslist#queries
-#
-# For IE 9-11 support, please remove 'not' from the last line of the file and adjust as needed
+
+# Googlebot uses an older version of Chrome
+# For additional information see: https://developers.google.com/search/docs/guides/rendering
+
+# For IE 9-11 support, remove '#' next to 'IE 9-11'.
 
 > 0.5%
 last 2 versions
 Firefox ESR
 not dead
-not IE 9-11
\ No newline at end of file
+# IE 9-11
+Chrome 41 # Googlebot
```

Comment:
@clydin had some good arguments for not including Chrome 41. // cc @alexeagle

Reply:
Disabling again by default.